### PR TITLE
Add Bazel support for ctc lib and Hook it up CI

### PR DESCRIPTION
**Pull request by tejal29**
This PR is based on  https://github.com/GoogleCloudPlatform/runtimes-common/pull/533

Please review https://github.com/GoogleCloudPlatform/runtimes-common/pull/551/commits/3d5dd0a8e1d68b6d0673d1646db7cb9bf367e4e5

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -112,3 +118,18 @@ docker_pull(
     registry = "gcr.io",
     repository = "gae-runtimes/php72_app_builder"
 )
+
+# Go Dependencies for Container Tool Command Library.
+# Import Go dependencies.
+go_repository(
```
Comment (dlorenc) [inline WORKSPACE]: I think if you use gazelle with mode external="vendored" you don't need these.

File: ctc_lib/BUILD [hunk 2]
```diff
@@ -1 +1,29 @@
...
+    ],
+)
+
+go_test(
+    name = "ctc_test",
+    srcs = glob(["*_test.go"]),
+    embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
+    deps = [
+        #":go_default_library",
```
Comment (tejal29) [inline ctc_lib/BUILD]: Remove this.

File: demo/ctc/cmd/root.go [hunk 3]
```diff
@@ -45,6 +44,7 @@ var RootCommand = &ctc_lib.ContainerToolCommand{
 }
 
 func Execute() {
+	ctc_lib.Version = "1.0.1"
 	RootCommand.Execute()
```
Comment (tejal29) [inline demo/ctc/cmd/root.go]: Change this to ctc_lib.Execute(RootCommand)

File: WORKSPACE (hunk 1 above)
Comment (tejal29) [reply WORKSPACE]: yeah i saw @aaron-prindle added after i started working on this. I will try gazelle.

Comment (tejal29) [reply WORKSPACE]: done

File: versioning/scripts/cloudbuild/BUILD.bazel [hunk 4]
```diff
@@ -11,6 +11,5 @@ go_library(
 go_binary(
     name = "cloudbuild",
     embed = [":go_default_library"],
-    importpath = "github.com/GoogleCloudPlatform/runtimes-common/versioning/scripts/cloudbuild",
```
Comment (tejal29) [inline versioning/scripts/cloudbuild/BUILD.bazel]: I am not sure why this got removed after running the gazelle tool.

**Pull request merged**